src/installing-deis/configuring-object-storage.md

```diff
@@ -20,6 +21,32 @@ The Deis components determine what object storage system to use via environment
 - `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` - The in-cluster Minio service
 - `DEIS_OUTSIDE_STORAGE_HOST` and `DEIS_OUTSIDE_STORAGE_PORT` - The external S3-compatible object storage system
 
+# Storing Credentials
+
+In the Deis V2 Beta release, all components read credentials from the filesystem, and we suggest that credentials are stored in [Kubernetes secrets](http://kubernetes.io/v1.1/docs/user-guide/secrets.html) and mounted to the appropriate location for the component. See the below list for the expected location for each component, and see [the deis-dev chart](https://github.com/deis/charts/tree/master/deis-dev) for examples of using and mounting secrets.
+
+- [builder](https://github.com/deis/builder)
+  - Key: `/var/run/secrets/object/store/access-key-id`
+  - Secret `/var/run/secrets/object/store/access-key-secret`
+- [slugbuilder](https://github.com/deis/slugbuilder)
+  - Key: `/var/run/secrets/object/store/access-key-id`
+  - Secret `/var/run/secrets/object/store/access-key-secret`
+- [slugrunner](https://github.com/deis/slugrunner)
+  - Key: `/var/run/secrets/object/store/access-key-id`
+  - Secret: `/var/run/secrets/object/store/access-key-secret`
+- [registry](https://github.com/deis/registry)
+  - Key: `/var/run/secrets/deis/registry/creds/accesskey`
+  - Secret: `/var/run/secrets/deis/registry/creds/secretkey`
+- [database](https://github.com/deis/postgres)
+  - Key: `/etc/wal-e.d/env/access-key-id`
+  - Secret: `/etc/wal-e.d/env/access-key-secret`
+
+# A Note on Google Cloud Storage
+
+As you may know Google Cloud Storage (GCS) can [interoperate with the S3 API](https://cloud.google.com/storage/docs/interoperability), and, if you choose to use Google Cloud Storage for object storage, you'll have to turn on this interoperability mode.
+
+If you choose to use Google Cloud Storage, set your `DEIS_OUTSIDE_STORAGE_HOST` environment variable to `storage.googleapis.com`, and follow [these instructions](https://cloud.google.com/storage/docs/migrating?hl=en_US#keys) to generate an S3 compatible access key ID and access key secret. Store these credentials just as you would if they were AWS S3 or Minio credentials (see the "Storing Credentials" section above).
```

Comment:
URL should be along with the scheme http or https 
https://github.com/deis/builder/blob/master/pkg/gitreceive/storage/endpoint.go#L25